packages/components/src/components/toggle-switch/toggle-switch.scss

```diff
@@ -23,7 +23,7 @@
         content: '';
         position: absolute;
         top: 6px;
-        left: 8px;
+        left: 4px;
```

Comment:
@artsemkavaleu-deriv  Please don't change the components package code, that's reusable code.

Reply:
sure, thx for notice!